cpp/src/lists/drop_list_duplicates.cu

```diff
@@ -533,7 +496,7 @@ std::unique_ptr<column> generate_output_offsets(size_type num_lists,
 
   // Generate offsets from sizes.
   // Given the example above, we will have new_offsets = [0, 3, 3, 3, 4, 7, 7, 9]
-  thrust::inclusive_scan(
+  thrust::exclusive_scan(
```

Comment:
This change is due to switching to using 0-starting labels from 1-starting labels.